File: Iniciante/1046.py
line1 = input().split()

hi = int(line1[0])
hf = int(line1[1])

h = hf-hi

if h < 1:
  h += 24
  print("O JOGO DUROU {} HORA(S)".format(h))

if h == 0:
  print("O JOGO DUROU 24 HORA(S)")

if hf > hi:
  print("O JOGO DUROU {} HORA(S)".format(h))

File: Iniciante/1046.js
let line1 = lines.shift().split(' ');

let hi = Number(line1.shift());
let hf = Number(line1.shift());

let h = hf-hi;

if (h < 1) {
  h += 24;
  console.log("O JOGO DUROU " + h + " HORA(S)");
}

if (h == 0) {
  console.log("O JOGO DUROU 24 HORA(S)");
}

if (hf > hi) {
  console.log("O JOGO DUROU " + h + " HORA(S)");
}
